test/it/uk/gov/hmrc/exports/base/IntegrationTestSpec.scala

```diff
@@ -17,20 +17,44 @@
 package uk.gov.hmrc.exports.base
 
 import com.codahale.metrics.SharedMetricRegistries
-import com.google.inject.AbstractModule
 import org.scalatest.concurrent.IntegrationPatience
-import play.api.inject.guice.GuiceableModule
-import stubs.WireMockRunner
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import play.api.Application
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.test.Injecting
+import stubs.ExternalServicesConfig.{Host, Port}
+import stubs.CustomsDeclarationsAPIConfig.{apiVersion, submitDeclarationServiceContext}
+import stubs.MockGenericDownstreamService
+import testdata.ExportsTestData
+import uk.gov.hmrc.exports.util.TestModule
+import uk.gov.hmrc.http.HeaderCarrier
 
-object IntegrationTestModule extends AbstractModule {
-  override def configure(): Unit = ()
+trait IntegrationTestSpec extends UnitSpec with GuiceOneAppPerSuite with Injecting with IntegrationPatience with MockGenericDownstreamService {
 
-  def asGuiceableModule: GuiceableModule = GuiceableModule.guiceable(this)
-}
+  SharedMetricRegistries.clear()
 
-trait IntegrationTestSpec extends UnitSpec with IntegrationPatience with WireMockRunner {
+  override implicit lazy val app: Application =
+    GuiceApplicationBuilder(overrides = Seq(TestModule.asGuiceableModule))
+      .configure(
+        Map(
+          "microservice.services.auth.host" -> Host,
+          "microservice.services.auth.port" -> Port,
+          "microservice.services.customs-data-store.host" -> Host,
+          "microservice.services.customs-data-store.port" -> Port,
```

Comment:
No key for `verified-email-path` ? It probably works because WireMock can be set up with any endpoint. And it gets some empty value.